### ISSUE TITLE
error: type 'Bundle' has no member 'module'

### ISSUE THREAD

**Issue by Bersaelor** [unsignedapps/swift-create-xcframework#33]
Replace this paragraph with a short description of the incorrect incorrect behavior. If this is a regression, please note the last version that the behavior was correct in addition to your current version. Please also include any information you have about the environment as this tool is very sensitive to Xcode and SDK versions.

**swift-create-xcframework version:** `1.3.1` 
**Swift version:** 
```
Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)
Target: x86_64-apple-darwin19.6.0
```
**Environment:**
```
Xcode: Version 12.4 (12D4e)
macOS: 10.15.7
```

### Checklist
- [x] I've searched for [existing GitHub issues](https://github.com/unsignedapps/swift-create-xcframework/issues)

### Steps to Reproduce

In Swift packages, a file is created that gives `Bundle` the `static var module: Bundle` variable, which can then be used in methods like:
```swift
extension String {
    var localized: String {
        return NSLocalizedString(self, tableName: nil, bundle: .module, value: "", comment: "")
    }
}
```

This works as advertised inside a Swift package. that is either imported into a host app or by clicking on a `Package.swift`.
The problem seems to be that when I use `generate-xcodeproj` or swift-create-xcframework, to create an XCFramework, the extension of `Bundle` is not provided and the code fails to compile.

Basically the same as described in this [SO thread](https://stackoverflow.com/questions/64407950/swift-package-manager-type-bundle-has-no-member-module-error)

The SO Thread also has the source code, for the SPM internally generated Bundle Extension file.

### Expected behavior

The swift package that compiles and works when used as a SP also compiles and works in `swift-create-xcframework`

### Actual behavior
```
/Users/konrad/Source/LCEyetest/Sources/LCEyetest/Eyetest.swift:99:84: error: type 'Bundle?' has no member 'module'
        guard let navigationController = UIStoryboard(name: "AcuityTest", bundle: .module)
```

### Hacky fix

By putting the SPM generated file (posted below) manually into the sources folder, swift-create-xcframework will succeed.
BUT, then the Swift package will no longer compile by itself (when used as a Swift package or opened via the Package.swift), as it will then have the following error:

<img width="988" alt="Screen Shot 2021-02-18 at 17 39 14 " src="https://user-images.githubusercontent.com/4517582/108389726-3ccc5780-7210-11eb-88c0-cf4cff0b744c.png">

```swift
import class Foundation.Bundle

private class BundleFinder {}

extension Foundation.Bundle {
    /// Returns the resource bundle associated with the current Swift module.
    static var module: Bundle = {
        let bundleName = "LCEyetest_LCEyetest"

        let candidates = [
            // Bundle should be present here when the package is linked into an App.
            Bundle.main.resourceURL,

            // Bundle should be present here when the package is linked into a framework.
            Bundle(for: BundleFinder.self).resourceURL,

            // For command-line tools.
            Bundle.main.bundleURL,
        ]

        for candidate in candidates {
            let bundlePath = candidate?.appendingPathComponent(bundleName + ".bundle")
            if let bundle = bundlePath.flatMap(Bundle.init(url:)) {
                return bundle
            }
        }
        fatalError("unable to find bundle named LCEyetest_LCEyetest")
    }()
}
```

**Comment by bok-** [comment]
Thanks for the bug report @Bersaelor. I think it should be possible to generate the above and reference it in the generated Xcode project we use internally. I'll check it out later today

**Comment by mthole** [comment]
@bok- I ran into this today and was happy to find this ticket. Any luck investigating?

**Comment by darecki** [comment]
I just encountered the same issue. It would be nice to have it fixed.
@Bersaelor @bok- @mthole have you found a way to make it work?

**Comment by acosmicflamingo** [comment]
I just encountered this bug :( Can this be reopened and fixed? :D

**Comment by bok-** [comment]
@acosmicflamingo Yep it can. I actually have recently completed the rebuild for this and am just testing it in a number of scenarios (including these bundle ones). I'll put out a pre-release version once that testing is done, would you be able to check if it fixes it in your usage?

**Comment by acosmicflamingo** [comment]
@bok- Oh wow that's excellent news!!! Yes I'd be absolutely happy to test it out for my use case whenever you are ready.

**Comment by acosmicflamingo** [comment]
@bok- Hi there! Thought I'd check in to see if you would like me to start testing out anything for you; I am still interested in this :)

**Comment by zladzeyka** [comment]
I am also still interested in it.

**Comment by sultan-arshi** [comment]
+1
any body have luck to bypass or it is fixed 
or any other library ?

**Comment by mohamed-3amer** [comment]
@bok- Please do you have any plans to push the fixes soon as you mentioned above?

**Comment by alex93133** [comment]
Any updates?(

**Comment by tmaly1980** [comment]
What's the solution here?

**Comment by bdrelling** [comment]
@bok- Echoing the question from Feb of last year -- do you have plans to push the fixes you mentioned anytime soon?

This issue should also probably remain Open while it's still an ongoing issue... if I saw this issue prior to pulling in this command, it would have saved me a lot of time.

I would call this out on the README as well because it's pretty critical for supporting Swift 5.3+. The majority of my packages include resources in some capacity.

Thanks for your hard work!
